packages/office-ui-fabric-react/src/components/__snapshots__/Facepile.Basic.Example.tsx.shot

```diff
@@ -207,7 +306,7 @@ exports[`Component Examples renders Facepile.Basic.Example.tsx correctly 1`] = `
               {
                 -moz-osx-font-smoothing: grayscale;
                 -webkit-font-smoothing: antialiased;
-                border-radius: 0px;
+                border-radius: 50%;
```

Comment:
Here's what I have dug up so far.

This does reflect the same styles from the scss. https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Facepile/Facepile.scss#L33

In the last iteration of this PR, @JasonGore gave this response on this specific issue "Tests will more accurately show JS styling, so that makes sense." https://github.com/OfficeDev/office-ui-fabric-react/pull/4998#pullrequestreview-128170797

Additionally, an inspection in the browser of the current live facepile example https://developer.microsoft.com/en-us/fabric#/components/facepile shows me if you remove the 50% border-radius from the facepile itembuttons, they become square.

Reply:
I will help take a look, thanks for the investigation thus far.

Reply:
I now understand what is happening. The previous snapshot was capturing a child component's getStyles. In this case, the child component is BaseButton - and the 16px belongs there.

The 16px was being overriden to 0px from CSS, but we didn't capture this in the snapshots in the past. Now that it is fully converted, it is captured :) 

So that's the rationalization process. And the way one would double check on all of these changes is to inspect the element pre-change to ensure it is 0px too (which you've done).

So in other words, this looks good :+1: